Doc/library/platform.rst

```diff
@@ -360,6 +360,34 @@ Android Platform
 
    .. versionadded:: 3.13
 
+.. _platform-cli:
+
+Command-line Usage
+------------------
```

Comment:
```suggestion
Command-line usage
------------------
```
Let's use sentence case for headings: https://devguide.python.org/documentation/style-guide/#capitalization

You can update the others in this file as well if you like, we already have a mix: the top level is sentence case and the subheadings are title case.

Reply:
done and thank you for link because i missed the documentation section in devguide :-)